src/SignInHelper.re

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
I'm trying to colocate this stuff to the function that consumes it.  If you find yourself needing it more than once, put it in the Types.re file and `open` that.

Eventually I'll have to think harder about whether we modularize the functions here or hide them behind an interface.  But whatever way we go, having them organized as above will make it easier to spot patterns, overlap, and cruft.

Reply:
That file is just getting so long! Hard to scroll between things when working through it. I have actually just been putting the one call, client, everything in one file until it works, the cp into Cognito.re...Is that bad?

Reply:
Frankly I tend to build really long files early in a project until I understand what abstractions are appropriate.  I'm not there yet, so I don't really care how long Cognito.re gets!  If you use vscode, you can open it twice to mitigate some of the scrolling pain:
![image](https://user-images.githubusercontent.com/2559043/65839273-e3891e80-e2d0-11e9-922a-b3bfe351de40.png)

Reply:
I noticed you have added `poolid` and `region` to the paramaters here:
https://github.com/pckilgore/reason-cognito/blob/f1b437a8eaafd19abafae9443069537bbce03ec0/src/Scratch.re

I need a `poolId` and `region` to keep going. Have one? We werent using it before and if I take them out now, it throws:

```
$ node src/Scratch.bs.js
[
  281060686,
  'User pool client 3vjshpa4lgf92nfisjrg9os21a does not exist.'
]
```
Have you tried the new api against the endpoint yet?

Reply:
Look at my most recent scratch file?

Reply:
I'm at work, but I'll double check when I get home.

Reply:
nothing new in the most recent scratch in dev branch in the repo. Same dummy params. Will check later when you get a chance to look at it.
https://github.com/pckilgore/reason-cognito/blob/f1b437a8eaafd19abafae9443069537bbce03ec0/src/Scratch.re#L1-L9